data_resource_api/app/exception_handler.py

```diff
@@ -0,0 +1,92 @@
+import json
+from brighthive_authlib import OAuth2ProviderError
+from data_resource_api.logging import LogFactory
+from flask import jsonify
+from werkzeug.exceptions import NotFound
+
+
+class DRApiError(Exception):
+    def __init__(self, message, status_code=400, errors=[]):
+        Exception.__init__(self)
+        self.message = message
+        self.errors = errors
+        self.status_code = status_code
+
+    def get_message(self):
+        resp = {}
+        resp['error'] = self.message
```

Comment:
I think this should be resp[`message`]. That's what I am seeing as the convention, e.g. [here](https://flask-restplus.readthedocs.io/en/latest/errors.html). But maybe everything should be wrapped inside an `error`, [like this](https://blog.restcase.com/rest-api-error-handling-problem-details-response/). Honestly, whomever ingests our APIs will figure it out, but maybe look into what's best practice....?

Reply:
Oh, I found another resource for this: https://dzone.com/articles/rest-api-error-handling-problem-details-response

Reply:
This is great feedback. I'm going to open this as a new issue and continue on.